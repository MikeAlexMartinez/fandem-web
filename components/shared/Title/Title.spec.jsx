import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import TitleComponent from './Title';

const classes = {
  root: {
    test: 'test',
  },
  content: {
    test: 'test',
  },
  title: {
    test: 'test',
  },
  controls: {
    test: 'test',
  },
};

describe('<Title />', () => {
  it('renders and matches the snapshot', () => {
    const wrapper = shallow(
      <TitleComponent classes={classes} title="Test Title">
        <div>Children</div>
      </TitleComponent>,
    );
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
