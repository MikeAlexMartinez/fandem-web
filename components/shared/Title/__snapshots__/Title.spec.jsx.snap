// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> renders and matches the snapshot 1`] = `
<div
  className="test"
>
  <div
    className="test flex row jc-sb ai-center"
  >
    <div
      className="test"
    >
      <WithStyles(Typography)
        variant="h3"
      >
        Test Title
      </WithStyles(Typography)>
    </div>
    <div
      className="test flex row jc-end ai-center"
    >
      <div>
        Children
      </div>
    </div>
  </div>
  <WithStyles(Divider) />
</div>
`;
